src/modules/repository/implementation/sequelize/repositories/paranet-asset-repository.js

```diff
@@ -0,0 +1,123 @@
+import Sequelize from 'sequelize';
+
+class ParanetAssetRepository {
+    constructor(models) {
+        this.sequelize = models.sequelize;
+        this.model = models.paranet_asset;
+    }
+
+    async createMissedParanetAssetRecord(missedParanetAsset, options) {
+        return this.model.create({ ...missedParanetAsset, isSynced: false }, options);
+    }
+
+    async createParanetSyncedAssetRecord(
+        blockchainId,
+        ual,
+        paranetUal,
+        publicAssertionId,
+        privateAssertionId,
+        sender,
+        transactionHash,
+        options,
+    ) {
+        return this.model.create(
+            {
+                blockchainId,
+                ual,
+                paranetUal,
+                publicAssertionId,
+                privateAssertionId,
+                sender,
+                transactionHash,
+                isSynced: true,
+            },
+            options,
+        );
+    }
+
+    async getCountOfMissedAssetsOfParanet(paranetUal, options = {}) {
+        return this.model.count({
+            where: {
+                paranetUal,
+                isSynced: false,
```

Comment:
This should also take into account retry to be less than constant and that updateAt  + delay > current time

Reply:
Something like that is laready done on line 58

Reply:
we use both queries for different reasons i think